pyproject.toml

```diff
@@ -25,6 +25,7 @@ dynamic = [
     "version",
 ]
 dependencies = [
+    "asvcore",
```

Comment:
How is the asvcore package on PyPI created? Is it from this repo?

Reply:
Currently it's over here:

https://github.com/HaoZeke/asvcore

If the PR and implementation looks good the next step would be to migrate it to the airspeed-velocity organization.

Reply:
Is there a design document somewhere with how these pieces fit together: what belongs in asvcore and what remains in asv? I am not sure I understand the division.